test/integration/custom-server/prod.spec.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+import * as url from 'url';
+import invariant from 'invariant';
+import { test } from '../../playwright/localTest';
+import { expectBasicPageContent } from '../backend-basic/shared';
+
+const currentDirectory = url.fileURLToPath(new URL('.', import.meta.url));
+
+test.use({
+  projectConfig: {
+    template: path.resolve(currentDirectory, '../backend-basic/fixture'),
+  },
+  customServerConfig: {
+    dev: false,
+    env: {
+      SECRET_BAZ: 'Some baz secret',
+    },
+  },
+});
+
+test('custom server production mode', async ({ context, customServer, page }) => {
```

Comment:
Should we extract the test content somewhere so it doesn't have to be written twice?